homeassistant/components/roborock/coordinator.py

```diff
@@ -73,8 +74,12 @@ async def _update_device_prop(self, device_info: RoborockHassDeviceInfo) -> None
     async def _async_update_data(self) -> dict[str, RoborockDeviceProp]:
         """Update data via library."""
         try:
-            for device_info in self.devices_info.values():
-                await self._update_device_prop(device_info)
+            asyncio.gather(
+                *[
```

Comment:
I think it is preferred to use a tuple instead of a list when a list is not needed.